packages/features/calendars/DestinationCalendarSelector.tsx

```diff
@@ -134,7 +131,7 @@ const DestinationCalendarSelector = ({
             <span className="min-w-0 overflow-hidden truncate whitespace-nowrap">
               {t("default_calendar_selected")}{" "}
               {queryDestinationCalendar.name &&
-                `(${queryDestinationCalendar?.integration} - ${queryDestinationCalendar.name})`}
+                `| ${queryDestinationCalendar.name} (${queryDestinationCalendar?.integrationTitle} - ${queryDestinationCalendar.primaryEmail})`}
```

Comment:
This format matches more closely to what it looks like when we select a calendar.
![image](https://user-images.githubusercontent.com/65426560/221072047-284aa6ea-f271-40fd-95b7-99ae85e91714.png)
![image](https://user-images.githubusercontent.com/65426560/221072083-f6412a54-fa31-44cb-9964-3c887e0b4395.png)